ports/openh264/vcpkg.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "openh264",
+  "version": "2.1.1",
```

Since you used the latest commit instead of a released version here, it might be better to use a date as version.

Please see https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#version and https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions
```suggestion
  "version-date": "2021-03-16",
```